src/components/my-app.js

```diff
@@ -288,18 +288,18 @@ class MyApp extends LitElement {
 
     switch(page) {
       case 'view1':
-        await import('../components/my-view1.js');
+        import('../components/my-view1.js');
         // Put code here that you want it to run every time when
         // navigate to view1 page and my-view1.js is loaded
         break;
       case 'view2':
-        await import('../components/my-view2.js');
+        import('../components/my-view2.js');
         break;
       case 'view3':
-        await import('../components/my-view3.js');
+        import('../components/my-view3.js');
         break;
       default:
-        await import('../components/my-view404.js');
```

Comment:
To be really consistent with `master`, `page = 'view404'` should be assigned here in the `default` case and `this._page = page` set after `switch`. (See #138 for background if interested.)